src/components/Paragraph.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+
+const Paragraph = ({ children, className }) => {
+  return (
+    <p
+      className={`text-2xl leading-relaxed text-gray-500 ${className} font-normal`}
+    >
```

Comment:
Convert this to single line

`<p className={`text-2xl leading-relaxed text-gray-500 ${className} font-normal`}>`

And add storybook item for this component.

Reply:
Done. Thanks